docs/UserGuide.md

```diff
@@ -123,11 +124,11 @@ Increases or decreases the number of times a student attended tutorials.
 
 Format: `attendance INDEX s/SIGN v/VALUE`
 
-* Adds attendance to the student at the specified INDEX
+* Adds attendance to the student at the specified INDEX.
 * The index refers to the index number shown in the displayed student list.
-* The index must be a positive integer 1, 2. 3, …
-* decrease attendance value if `SIGN` is '-' and increase attendance value if `SIGN` is '+'
-* increase or decrease the specific student's attendance by `VALUE`
+* The index must be a positive integer 1, 2. 3, ….
+* decrease attendance value if `SIGN` is '-' and increase attendance value if `SIGN` is '+'.
+* increase or decrease the specific student's attendance by `VALUE`.
```

Comment:
I'm sorry, the command format has been changed. I will make sure to change it to the correct format at my side afterwards.